ReportSimulationOutput/measure.rb

```diff
@@ -2882,6 +2882,12 @@ def get_object_output_variables_by_key(model, object, sys_id, class_name)
 
       if object.to_ElectricLoadCenterStorageLiIonNMCBattery.is_initialized
         return { RT::Battery => ['Electric Storage Charge Fraction'] }
+
+      elsif object.to_EnergyManagementSystemOutputVariable.is_initialized
+        if object.name.to_s.include? Constants.ObjectNameBatteryLossesAdjustment(nil)
+          return { RT::Battery => [object.name.to_s] }
+        end
```

Comment:
This is the primary bugfix. Need to ensure we always create an output variable for the battery losses -- previously the code would _happen_ to create an output variable if you requested hourly timeseries outputs, but not without the request.